src/Search.php

```diff
@@ -7122,7 +7122,8 @@ public static function giveItem(
                             }
                         }
                         if ($totaltime != 0) {
-                            $percentage  = round((100 * $currenttime) / $totaltime);
+                            $waitingtime = $item->fields['sla_waiting_duration'];
```

Comment:
The `$waitingtime` value should be 0 if there is no SLA defined. Also, the `sla_waiting_duration` is only related to the TTR SLA and should not have any impact on TTO and/or OLA. You should define this variable in the previous if/else blocks.
```php
if ($item->isField($slaField) && $item->fields[$slaField] != 0) { // Have SLA
    ...
    $waitingtime = $slaField === 'slas_id_ttr' ? $item->fields['sla_waiting_duration'] : 0;
    ... 
} else {
    ...
    $waitingtime = 0;
    ...
}
```